modules/dynamic-plugins/rhdh-supported-plugins.adoc

```diff
@@ -15,75 +15,219 @@ Additional detail on how Red Hat provides support for bundled community dynamic
-a|
+
 |Disabled
 
-|Analytics Provider Segment  |@janus-idp/backstage-plugin-analytics-provider-segment |Frontend |1.5.0 |Production
-|./dynamic-plugins/dist/janus-idp-backstage-plugin-analytics-provider-segment
-a|
-`SEGMENT_WRITE_KEY`
+|JFrog Artifactory  - Frontend |https://npmjs.com/package/@janus-idp/backstage-plugin-jfrog-artifactory/v/1.5.0[@janus-idp/backstage-plugin-jfrog-artifactory] |1.5.0 
+|./dynamic-plugins/dist/janus-idp-backstage-plugin-jfrog-artifactory
 
-`SEGMENT_TEST_MODE`
+|Disabled
 
-|Enabled
+|Nexus Repository Manager  - Frontend |https://npmjs.com/package/@janus-idp/backstage-plugin-nexus-repository-manager/v/1.7.0[@janus-idp/backstage-plugin-nexus-repository-manager] |1.7.0 
+|./dynamic-plugins/dist/janus-idp-backstage-plugin-nexus-repository-manager
 
-|Argo CD  |@janus-idp/backstage-plugin-argocd |Frontend |1.2.3 |Production
-|./dynamic-plugins/dist/janus-idp-backstage-plugin-argocd
-a|
 |Disabled
 
-|Argo CD  |@roadiehq/backstage-plugin-argo-cd |Frontend |2.6.5 |Production
-|./dynamic-plugins/dist/roadiehq-backstage-plugin-argo-cd
-a|
-|Disabled
+|Scaffolder Relation Processor  - Backend |https://npmjs.com/package/@janus-idp/backstage-plugin-catalog-backend-module-scaffolder-relation-processor/v/1.1.0[@janus-idp/backstage-plugin-catalog-backend-module-scaffolder-relation-processor] |1.1.0 
+|./dynamic-plugins/dist/janus-idp-backstage-plugin-catalog-backend-module-scaffolder-relation-processor-dynamic
 
-|Argo CD  |@roadiehq/backstage-plugin-argo-cd-backend |Backend |3.0.2 |Production
-|./dynamic-plugins/dist/roadiehq-backstage-plugin-argo-cd-backend-dynamic
-a|
-`ARGOCD_USERNAME`
+|Enabled
 
-`ARGOCD_PASSWORD`
+|ServiceNow  - Backend |https://npmjs.com/package/@janus-idp/backstage-scaffolder-backend-module-servicenow/v/1.5.0[@janus-idp/backstage-scaffolder-backend-module-servicenow] |1.5.0 
+|./dynamic-plugins/dist/janus-idp-backstage-scaffolder-backend-module-servicenow-dynamic
 
-`ARGOCD_INSTANCE1_URL`
+`SERVICENOW_BASE_URL`
 
-`ARGOCD_AUTH_TOKEN`
+`SERVICENOW_USERNAME`
 
-`ARGOCD_INSTANCE2_URL`
+`SERVICENOW_PASSWORD`
 
-`ARGOCD_AUTH_TOKEN2`
+|Disabled
+
+|SonarQube  - Backend |https://npmjs.com/package/@janus-idp/backstage-scaffolder-backend-module-sonarqube/v/1.5.0[@janus-idp/backstage-scaffolder-backend-module-sonarqube] |1.5.0 
+|./dynamic-plugins/dist/janus-idp-backstage-scaffolder-backend-module-sonarqube-dynamic
 
 |Disabled
 
-|Argo CD  |@roadiehq/scaffolder-backend-argocd |Backend |1.1.27 |Community Support
+|===
+
+=== Community plugins
+
+Red Hat provides a select number of community-supported plugins, available for customers to configure and enable. These included community plugins are exported by Red Hat as plugins, and are provided with support scoped per Technical Preview terms. 
+
+Additional details on how Red Hat provides support for bundled community dynamic plugins are available on the https://access.redhat.com/policy/developerhub-support-policy[Red Hat Developer Support Policy] page.
```

Comment:
@nickboldt, I think @Gerry-Forde and @Preeticp had a plan to include the support policy with a note.
Gerry, could you please share that note?

Reply:
do you mean the note in the 1.2 docs? https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html-single/configuring_plugins_in_red_hat_developer_hub/index#rhdh-supported-plugins 

![image](https://github.com/user-attachments/assets/5003b1e1-7ab7-4d96-a869-a9942dd4e7e3)

Reply:
Please apply the notes as per https://docs.google.com/document/d/1F6ctaXcUis4ngcxGYhdPmdY23iQzQgz1v4lObn6WqZc/edit